diff -up tree-1.7.0/tree.c.no-color-by-default tree-1.7.0/tree.c
--- tree-1.7.0/tree.c.no-color-by-default	2014-04-25 16:18:58.034525513 +0100
+++ tree-1.7.0/tree.c	2014-04-25 16:20:04.953855799 +0100
@@ -94,7 +94,8 @@ int main(int argc, char **argv)
   q = p = dtotal = ftotal = 0;
   aflag = dflag = fflag = lflag = pflag = sflag = Fflag = uflag = gflag = FALSE;
   Dflag = qflag = Nflag = Qflag = Rflag = hflag = Hflag = siflag = cflag = FALSE;
-  noindent = force_color = nocolor = xdev = noreport = nolinks = reverse = FALSE;
+  noindent = force_color = xdev = noreport = nolinks = reverse = FALSE;
+  nocolor = TRUE;
   ignorecase = matchdirs = dirsfirst = inodeflag = devflag = Xflag = Jflag = FALSE;
   duflag = pruneflag = FALSE;
   flimit = 0;
